codeowners/src/github.rs

```diff
@@ -9,6 +9,11 @@ use std::{
     str::FromStr,
 };
 
+#[cfg(feature = "pyo3")]
+use pyo3::prelude::*;
+#[cfg(feature = "pyo3")]
+use pyo3_stub_gen::derive::{gen_stub_pyclass, gen_stub_pymethods};
+
 use crate::{FromPath, FromReader, OwnersOfPath};
```

Comment:
We can probably fix the imports here a little so they are grouped by `std`, then regular crates, then `crate::blah` crates